scripts/migration/physics_service_rename.py

```diff
@@ -0,0 +1,36 @@
+import os
+from pathlib import Path
+
+if __name__ == "__main__":
```

Comment:
suggestion: It would be great to add a comment block to the top of this file to describe the intent of the file, how to use it and what it does/did

Reply:
+1

Reply:
Good call, added.